internal/indexer/definitions/torrentday.yaml

```diff
@@ -0,0 +1,58 @@
+---
+#id: torrentday
+name: TorrentDay
+identifier: td
+description: TorrentDay (TD) is a private torrent tracker for GENERAL.
+language: en-us
+urls:
+  - https://www.torrentday.com
+privacy: private
+protocol: torrent
+supports:
+  - irc
+  - rss
+source: custom
+settings:
+  - name: passkey
+    type: secret
+    label: Passkey
+    help: "Check the tracker tab in your torrent client to find your passkey"
+
+irc:
+  network: TorrentDay
+  server: irc.torrentday.com
+  port: 6697
+  tls: true
+  channels:
+    - "#td.announce"
+  announcers:
+    - TD_Announce
+  settings:
+    - name: nickserv.account
+      type: text
+      required: false
+      label: NickServ Account
+      help: NickServ account. Make sure to group your user and bot. Eg. username_autodl
+    - name: nickserv.password
+      type: secret
+      required: false
+      label: NickServ Password
+      help: NickServ password
+
+parse:
+  type: single
+  lines:
+    - test:
+        - "[Movie/4K] This is a Movie 2160p UHD BluRay x265-TEST - https://www.torrentday.com/details.php?id=0000000 - 5.83 GB"
+      pattern: '\[(.*?)\] (.*?) - (https?://.*/).*id=(\d+) - (.*)'
+      vars:
+        - category
+        - torrentName

```

Comment:
With freeleech in the pattern, it's now the third one. So add `- freeleech` below `- torrentName`.

Reply:
have updated the regex accordingly and added freeleech in 3rd position